### DISCUSSION TITLE
Help with image source link

### DISCUSSION THREAD

**Discussion by gurds91** [holger1411/understrap#1799] [Q&A]
Hi guys, on one of the pages of a site I am creating using Understrap I wanted to do a bootstrap carousel for images
the template for this page is called "certifications-temp.php" that I created based off a full width page and is in page-templates folder. The code below is essentially for certification-temp.php

the custom template file I created in my understrap child theme is "certifications-temp.php" and is located in page-templates folder.

I used following code for carousel:

<div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
  <div class="carousel-inner">
    <div class="carousel-item active">
      <img src="<?php echo get_template_directory_uri(); ?>/images/SW_html5_foundations.png"class="d-block w-100" alt="html-certification">
    </div>
    <div class="carousel-item">
      <img src="..." class="d-block w-100" alt="css-certification">
    </div>
    <div class="carousel-item">
      <img src="..." class="d-block w-100" alt="js-certification">
    </div>
    <div class="carousel-item">
      <img src="..." class="d-block w-100" alt="php-certification">
    </div>
  </div>
  <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="prev">
    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Previous</span>
  </button>
  <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="next">
    <span class="carousel-control-next-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Next</span>
  </button>
  <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleControls" data-bs-slide="next">
    <span class="carousel-control-next-icon" aria-hidden="true"></span>
    <span class="visually-hidden">Next</span>
  </button>
</div>

As you can see for img source for first image I tried  <img src="<?php echo get_template_directory_uri(); ?>/images/SW_html5_foundations.png"class="d-block w-100" alt="html-certification">

but it didn't work, images are in a folder called "images" in my theme folder and image in question is called "SW_html5_foundations.png"
![issues with code trying to get image src right](https://user-images.githubusercontent.com/107938466/176563712-35afba19-a367-45d8-9715-09895d34b40f.png)

Please help to fix this image source so that it works.

**Comment by IanDelMar** [comment]
What should we be seeing here?

![image](https://user-images.githubusercontent.com/42134098/176586556-63474b34-936f-4131-9804-ac8bdf2c4c8c.png)

If this is code, please use 
<pre>```
paste your code here
```</pre>

Have you tried to add the missing space before `class="d-block w-100"`?

**Reply by IanDelMar** [reply]
@gurds91 Were you able to resolve this issue?

**Reply by gurds91** [reply]
yeah, I managed to sort it by hosting the image online and linking to it.
